src/components/Player/Player.jsx

```diff
@@ -21,63 +26,82 @@ import {
   CountsPage,
   TrendsPage,
   MMRPage,
-  playerPages,
 } from './Pages';
 
-const playerPagesMapped = (accountId) => playerPages.map(({ name, ...rest }) => ({
-  ...rest,
-  route: `/players/${accountId}/${name.toLowerCase()}`,
-  label: name,
-}));
+const playerPages = [{
```

Comment:
putting this configration object into Player.jsx is also something I would like to avoid. I don't mind the switch case because it's essentially routing, and it's very intuitive what the code is doing. A big configuration object feels worse. Also, there is no reason to put `()` around all your return statements with jsx. It's just something that's needed for multiline jsx (and that may not even be necessary, but makes it easier to read).